### DISCUSSION TITLE
Call to FundMe.getVersion errored: Error occurred: revert.

### DISCUSSION THREAD

**Discussion by cbc8181** [Cyfrin/foundry-full-course-cu#2113] [Q&A]
Hello guys, I tried to call getVersion method, but it always prompt this error:
[call]from: 0x5B38Da6a701c568545dCfcB03FcB875f56beddC4to: FundMe.getVersion()data: 0x0d8...e6e2c
Debug
call to FundMe.getVersion errored: Error occurred: revert.

revert
	The transaction has been reverted to the initial state.
Note: The called function should be payable if you send value and the value you send should be less than your current balance.
You may want to cautiously increase the gas limit if the transaction went out of gas.

```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.18;

import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";

contract FundMe{
     uint256 public constant MINIMUM_USD = 5 * 10 ** 18;
    address[] public funders;
    mapping(address=>uint256) public addressToAmountFunded;

    function getVersion() public view returns (uint256) {
         AggregatorV3Interface priceFeed = AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306);
         return priceFeed.version();
    }
}
```

![image](https://github.com/user-attachments/assets/f092cfa8-d1e9-40eb-bf36-9eaeda116319)

**Comment by EngrPips** [comment]
It seems like you figured this out. Keep crushing it.
